test/mailers/previews/new_volunteer_application_mailer_preview.rb

```diff
@@ -0,0 +1,8 @@
+# Preview all emails at http://localhost:3000/rails/mailers/new_volunteer_application_mailer
```

Comment:
I know it's best to cut out comments. I would like to keep this for the url. I'm not sure I'll remember where to go otherwise.